### DISCUSSION TITLE
[Projects Blueprints] How can I set the git repository default visibility

### DISCUSSION THREAD

**Discussion by frengooo** [mia-platform/community#562] [Q&A]
### What product are you having troubles with?

Console

### What Console version are you using?

v13.0.5

### Description

Hi guys,
I have a project blueprint created on my company, attached to an Azure DevOps provider.
The blueprint works but, during the project creation phase, the Console always suggest me the "Public" visibility:
<img width="437" alt="Screenshot 2024-10-17 at 12 28 01" src="https://github.com/user-attachments/assets/e825aed0-41d5-47c0-991c-9f09d0737667">

Is there a way to set "Private" as default?

Thanks.

**Comment by davidebianchi** [comment]
Hi @silversoul93!
It is not possible at the moment to override the default visibility, but it is an interesting feature for sure! @jacoposte can we add this as small improvement?

**Reply by jacoposte** [reply]
We already have a similar activity in the product backlog to set the preferred visibility for each provider, although we do not currently have it expressly planned. 
I recommend following any updates on the release notes and the official roadmap!
